src/background/services/wallet/handlers/avalanche_sendTransaction.ts

```diff
@@ -305,7 +308,23 @@ export class AvalancheSendTransactionHandler extends DAppRequestHandler<
 
         onSuccess(res.txID);
       }
+
+      const duration = measurement.end();
+      this.analyticsServicePosthog.captureEncryptedEvent({
```

Comment:
We should add a `waitForTx` method to the Avalanche provider and use that here to have meaningful metrics IMO.